modules/social_features/social_mentions/social_mentions.module

```diff
@@ -9,7 +9,6 @@ use Drupal\Core\Form\FormStateInterface;
 use Drupal\comment\CommentInterface;
 use Drupal\Core\Url;
 use Drupal\Component\Serialization\Json;
-use Drupal\mentions\Entity\MentionsType;
```

Comment:
Might be best to add these changes in a separate PR, because this will likely move out of this repo

Reply:
I've created a separate PR https://github.com/goalgorilla/open_social/pull/3062

Reply:
But I left these changes in this PR to make sure that all tests will pass.